code/modules/food_and_drink/snacks.dm

```diff
@@ -424,6 +428,10 @@
 				src.food_effects = list()
 			..()
 			src.food_effects = food_effects_pre
+			if (!iswerewolf(M))
```

Comment:
I'd rather invert this

Reply:
fixed poorly written duplicate, put it in with the previous condition with pugs